internal/bminventory/inventory.go

```diff
@@ -2510,7 +2514,7 @@ func (b *bareMetalInventory) checkFileForDownload(ctx context.Context, clusterID
 	switch fileName {
 	case kubeconfig:
 		err = cluster.CanDownloadKubeconfig(&c)
-	case manifests.ManifestFolder:
+	case manifests.ManifestFolder, "discovery.ign":
 		// do nothing. manifests can be downloaded at any given cluster state
```

Comment:
Can't discovery.ign be downloaded in the following states: installing, finalizing, installed, installing-pending-user-action?

Reply:
The discovery ignition will be created before install starts. And if someone needs it to create a discovery iso then we can't possibly have started install.

Reply:
Right, mixed up my ignitions :)